scripts/zkai_d64_commitment_consistency_method_probe.py

```
@@ -485,11 +370,17 @@ def validate_probe(payload: dict[str, Any]) -> None:
         "activation_lookup_commitment": fixture["statement"]["activation_lookup_commitment"],
+        "proof_native_parameter_commitment": fixture["statement"]["proof_native_parameter_commitment"],
     }
     if payload["source_fixture"] != expected_source:
         raise CommitmentConsistencyProbeError("source fixture drift")
 
     expected_manifest = proof_native_parameter_manifest(FIXTURE.evaluate_reference_block())
+    if (
+        payload["source_fixture"]["proof_native_parameter_commitment"]
+        != payload["proof_native_parameter_manifest"]["proof_native_parameter_commitment"]
+    ):
+        raise CommitmentConsistencyProbeError("proof-native parameter commitment mismatch")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)